nodeodm/api_client.py

```diff
@@ -1,35 +1,41 @@
 """
-A wrapper around Bravado to communicate with a node-OpenDroneMap node.
+An interface to node-OpenDroneMap's API
+https://github.com/pierotofy/node-OpenDroneMap/blob/master/docs/index.adoc
 """
-from bravado.client import SwaggerClient
-from bravado.exception import HTTPError
-from requests import ConnectionError
+import requests
+import mimetypes
+import json
+import os
 
 class ApiClient:
-    def check_client(func):
-        def check(self, *args, **kwargs):
-            """
-            Makes sure that the client has been instantiated. 
-            Sometimes this will fail (rest endpoint might be offline), 
-            so we need to handle it gracefully...
-            """
-            if not hasattr(self, 'client'):
-                try:
-                    self.client = SwaggerClient.from_url('http://{}:{}/swagger.json'.format(self.host, self.port))
-                except (ConnectionError, HTTPError) as err:
-                    return None
-                    
-            return func(self, *args, **kwargs)
-        return check
-
     def __init__(self, host, port):
         self.host = host
         self.port = port
 
-    @check_client
+    def url(self, url):
+        return "http://{}:{}{}".format(self.host, self.port, url)
```

Comment:
Use `urlparse.urlunparse` instead to prevent the port number from showing up if it's the default (and to support https if necessary)?